api/systemIntegrations/post.js

```diff
@@ -199,6 +199,17 @@ function _validateMasterIntegrationFields(bag, next) {
 
   var errors = [];
 
+  _.each(bag.reqBody.data,
+    function (value, key) {
+      if (!_.findWhere(bag.masterIntegrationFields, {name: key})) {
+        return next(
+          new ActErr(who, ActErr.OperationFailed,
+            util.format('Invalid field: %s', key))
```

Comment:
Lets make it more clearer.
`Can not save <propertyName> as there is no masterIntegraitonField named <propertyName>.`